android/README.md

```diff
@@ -0,0 +1,82 @@
+# IronOxide-Android
+
+[![javadoc.io](https://javadoc.io/badge2/com.ironcorelabs/ironoxide-java/javadoc.io.svg)](https://javadoc.io/doc/com.ironcorelabs/ironoxide-java)
+
+## Using IronOxide-Android
+
+In order to make calls to the IronCore service, you must add a dependency for ironoxide-android, load the library, and provide authentication.
+
+### Adding a dependency
+
+IronOxide-Android is published to a custom maven repository. To pull from this repository, add the following to your project's "build.gradle":
+
+```java
+allprojects {
+    repositories {
+        maven {
+            url "https://oss.sonatype.org/content/repositories/snapshots"
+        }
+    }
+}
+```
```

Comment:
We publish to sonatype, but that gets transferred to Maven Central. Shouldn't we direct people to the main releases? I'd expect those to be on the default repositories list already, so we might not need this note at all.

Reply:
Looks like you're right! Something I had read made me think this was required, but taking it out of the Example App has no effect, so great!